apollo-cache-interceptor/src/commonMain/kotlin/com/apollographql/apollo3/interceptor/cache/CacheExecutionContext.kt

```diff
@@ -61,11 +60,10 @@ fun <D : Operation.Data> ApolloQueryRequest.Builder<D>.fetchPolicy(policy: Fetch
   addExecutionContext(policy)
 }
 
-val <D : Operation.Data> Response<D>.fromCache
-  get() = executionContext[CacheInfo]?.fromCache ?: throw IllegalStateException("ApolloGraphQL: no CacheInfo")
+val <D : Operation.Data> Response<D>.isFromCache
+  get() = requireNotNull(executionContext[CacheInfo]?.isFromCache)
```

Comment:
should we fallback to `false` instead throwing exception? Not sure if this is a critical issue to not having cache info context?

Reply:
👍  we could also add a better error message but defaulting to `false` makes sens